biorxiv/article_distances/03_biorxiv_corrected_published_rate.py

```diff
@@ -292,3 +292,82 @@
 g.save("output/figures/publication_rate_rerun.svg")
 g.save("output/figures/publication_rate_rerun.png", dpi=250)
 print(g)
+
+# # Plot Publication Rate
```

Comment:
Looks like rate = published/posted = number of papers published that month / ??? I guess I'm not sure what the posted means?

Reply:
Also is your label where the data is coming from? So you have different data sources for different time points?

Reply:
>Looks like rate = published/posted = number of papers published that month / ??? I guess I'm not sure what the posted means?

So posted means the date at which bioRxiv accepts a preprint and hosts it on their site. We are looking for the amount of preprints that are published compared to the amount of preprints posted for each month.

>Also is your label where the data is coming from? So you have different data sources for different time points?

Yes. The light blue is recapitulating a study that was previously done looking at the publication rate of preprints. The dark blue is our updated rate to show how publication changed over time. (Can't freeze papers from being published, so we provided an updated plot.) The dark green plot is our suggested correction when one accounts for missing links within bioRxiv as the matching process isn't the best.